docs/credentials.md

```diff
@@ -0,0 +1,131 @@
+<h1 align="center">Using Credentials with the OADP Operator</h1>
+
+
+### Creating a Secret
+
+- Use command `oc create secret generic <SECRET_NAME> --namespace openshift-adp --from-file cloud=<CREDENTIALS_FILE_PATH>`
+
+
+### Credentials for BackupStorageLocation:
+
+- If the secret name is not specified, OADP will use the default value.
+    - AWS default secret name is `cloud-credentials`
+    - GCP default secret name is `cloud-credentials-gcp`
+    - Azure default secret name is `cloud-credentials-azure`
+
+- If the `credential` spec has a custom name, this name will be used by the
+registry, and the default name will not be expected. An example is shown below:
+
+```
+spec:
+  ...
+  backupLocations:
+    - name: default
+      velero:
+        provider: aws
+        default: true
+        objectStorage:
+          bucket: my-bucket
+          prefix: my-prefix
+        config:
+          region: us-east-1
+          profile: "default"
+        credential:
+          name: my-custom-name
+          key: cloud
+```
+
+    
+### Credentials for VolumeSnapshotLocation:
+
+- `VolumeSnapshotLocation` will **always** expect the default secret name to exist, 
+which are described above.
+
+- To use seperate credentials for `BackupStorageLocation` and `VolumeSnapshotLocation`, 
```

Comment:
@eemcmullan I think I said this to you yesterday and I did forget one thing here... a user can actually take advantage of `profile` config key to use one secret with separate credentials. This only works for the AWS plugin but good to call out.

So one secret:
```
[backupStorage]
aws_access_key_id=...
aws_secret_access_key=...
[volumeSnapshot]
aws_access_key_id=...
aws_secret_access_key=...
```

Then the BSL:
```
     ...
     profile: "backupStorage"
```

and under snapshotlocations set it to the other profile

Reply:
@dymurray updated